strapi/strapi
# feature(content-releases): Add update release service/endpoint

### What does it do?

Add the new service/controller to update a release. For now it only works to edit the release's name.

### How to test it?

Go to postman or any tool you use to test API:
1. You should have permissions to edit a release.
2. Make a PUT request to `/content-releases/:releaseId`
3. On the body you should pass a JSON object with `name`, name should be a string.
4. It should return your Release updated
